cmd/gator/test/test.go

```diff
@@ -30,14 +30,17 @@ const (
   cat manifest.yaml | gator test
 
   # Output structured violations data
-  gator test --filename="manifest.yaml" --output=json`
+  gator test --filename="manifest.yaml" --output=json
+
+  Note: The alpha "gator test" has been renamed to "gator verify"`
```

Comment:
nit: might want to add what this command does vs. the old command so users don't think this is just a deprecated name and avoid looking at `gator test` altogether.